### PR TITLE
[MX-160] Prevent crash when camera roll permission denied

### PR DESCRIPTION
**Pull request by ds300**
https://artsyproduct.atlassian.net/browse/MX-160

### CODE REVIEW

File: src/lib/Components/Consignments/Screens/SelectFromPhotoLibrary.tsx [hunk 1]
```diff
@@ -52,9 +63,8 @@ export default class SelectFromPhotoLibrary extends React.Component<Props, State
     }
   }
 
-  loadPhotos() {
-    // TODO: Need to update the RN 0.48 types on DT
-    const fetchParams: any = {
+  async loadPhotos() {
+    const fetchParams: CameraRoll.GetPhotosParams = {
```
Comment (ashfurrow) [inline src/lib/Components/Consignments/Screens/SelectFromPhotoLibrary.tsx]: 🎉

File: src/lib/Components/Consignments/Screens/SelectFromPhotoLibrary.tsx [hunk 2]
```diff
@@ -188,12 +202,7 @@ export default class SelectFromPhotoLibrary extends React.Component<Props, State
       <Theme>
         <ProvideScreenDimensions>
           <BottomAlignedButton onPress={this.doneTapped} buttonText="Done">
-            <ScrollView
-              style={{ flex: 1 }}
-              scrollsToTop={true}
-              onScroll={this.onScroll.bind(this)}
```
Comment (ashfurrow) [inline src/lib/Components/Consignments/Screens/SelectFromPhotoLibrary.tsx]: I still haven't quite internalized the distinction between class functions and class properties that _are_ functions. So I see we changed `onScroll`:

```diff
-  onScroll(e) {
+  onScroll = (e: NativeSyntheticEvent<NativeScrollEvent>) => {
```

Is there a strong preference for one or the other, generally? Or does it depend on a case-by-case basis? Seems like not having to `.bind(this)` all the time would make `onScroll = ` preferred. I'd appreciate any clarity around nomenclature that I've gotten wrong, too 🙇

Comment (ds300) [reply src/lib/Components/Consignments/Screens/SelectFromPhotoLibrary.tsx]: > Seems like not having to .bind(this) all the time would make onScroll = preferred.

yep. Forgetting .bind is something i've seen cause subtle bugs occasionally.

**Comment by artsyit** [comment]
:rocket: PR was released in v1.21.39 :rocket:

**Pull request merged**